corehq/apps/hqwebapp/management/commands/complete_bootstrap5_report.py

```diff
@@ -0,0 +1,147 @@
+from django.core.management import BaseCommand
+
+from corehq.apps.hqwebapp.utils.bootstrap.paths import get_bootstrap5_path
+from corehq.apps.hqwebapp.utils.bootstrap.git import apply_commit, get_commit_string
+from corehq.apps.hqwebapp.utils.bootstrap.reports.progress import (
+    get_migrated_reports,
+    get_migrated_filters,
+    mark_filter_as_complete,
+    mark_report_as_complete,
+    get_migrated_filter_templates,
+    mark_filter_template_as_complete,
+)
+from corehq.apps.hqwebapp.utils.bootstrap.reports.stats import (
+    get_report_class,
+    get_bootstrap5_reports,
+)
+from corehq.apps.hqwebapp.utils.management_commands import get_confirmation
+
+
+class Command(BaseCommand):
+    help = "This command helps mark reports and associated filters (and their templates) as migrated."
+
+    def add_arguments(self, parser):
+        parser.add_argument('report_class_name')
+
+    def handle(self, report_class_name, **options):
+        self.stdout.write("\n\n")
+        report_class = get_report_class(report_class_name)
+        if report_class is None:
+            self.stdout.write(self.style.ERROR(
+                f"Could not find report {report_class_name}. Are you sure it exists?"
+            ))
+            return
```

Comment:
Nit: Is `stderr` preferable to `stdout`, since we're ending execution with an error? Better yet, it seems like Django says the preferred way to handle this is with [CommandError](https://docs.djangoproject.com/en/5.1/howto/custom-management-commands/#command-exceptions)?

Reply:
this is how we report error messages in the other tools. i'm not really keen on doing extra work to change it here.